src/PlaywrightSharp.Chromium/ChromiumExecutionContext.cs

```diff
@@ -76,7 +76,19 @@ public async Task<T> EvaluateAsync<T>(FrameExecutionContext context, bool return
 
         public Task ReleaseHandleAsync(JSHandle handle) => ReleaseObjectAsync(_client, handle.RemoteObject);
 
-        private static object ValueFromUnserializableValue(RemoteObject remoteObject, string unserializableValue)
+        public string HandleToString(IJSHandle handle, bool includeType)

```

Comment:
could we also override the native `ToString` method?

Reply:
We can't do that. It's a browser implementation.